radio/src/targets/taranis/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ option(GHOST "Ghost TX Module" ON)
 option(INTERNAL_MODULE_PPM "Support for PPM internal module" OFF)
 option(AUTOUPDATE "Auto update internal chips from SD" OFF)
 option(BIND_KEY "Enable bind button" OFF)
+option(BLUETOOTH "FrSky BT module support" NO)
```

Comment:
Minor beauty tip: could be uniformly "OFF" (instead of also working, but differing "NO" to match the other options).

Reply:
yes and no, later it says NO, not OFF in existring code, I have simply matches that. We could standardise on one or the other, there seems to be no consistency; but thats far beyond the scope of this